src/core/lender/assessor/default.sol

```diff
@@ -20,6 +20,8 @@ import "tinlake-math/interest.sol";
 
 contract DefaultAssessor is BaseAssessor, Interest {
 
+    constructor(uint tokenAmountForONE) BaseAssessor(tokenAmountForONE) public {}
+
     // accrueTrancheInterest can implement different interest models
     function accrueTrancheInterest(address tranche_) public returns (uint) {
```

Comment:
should we also do the same address check here?

Reply:
I will add it